src/BlazorWebView/src/Maui/iOS/BlazorWebViewHandler.iOS.cs

```diff
@@ -63,7 +64,8 @@ protected override WKWebView CreatePlatformView()
 				new NSString(BlazorInitScript), WKUserScriptInjectionTime.AtDocumentEnd, true));
 
 			// iOS WKWebView doesn't allow handling 'http'/'https' schemes, so we use the fake 'app' scheme
-			config.SetUrlSchemeHandler(new SchemeHandler(this), urlScheme: "app");
+			if (OperatingSystem.IsIOSVersionAtLeast(11))
```

Comment:
The warning is `'WKWebViewConfiguration.SetUrlSchemeHandler(IWKUrlSchemeHandler?, string)' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and later.`, 

Compared to unsupported warning, warnings for support would/should mean the API does not work in earlier versions, therefore I have taken more aggressive actions for platform support warnings. Though it is possible that the API annotated incorrectly, I do not know, if that is the case then the annotation should be fixed. If annotation is correct then the code owners should suggest with what we should do if isn't it iOS 11.

Reply:
Understood. I think the code change here is incorrect because this feature will simply fail in weird ways on iOS version < 11. It's possible that we just want to block it entirely, but then we would have to block it some other way (and not need this check here).

I logged https://github.com/dotnet/maui/issues/6078 to consider what BlazorWebView should do here.

Reply:
@buyaa-n - do you know how we can specify the minimum supported iOS (or whatever platform) that the analyzer should check? We're discussing whether we should have MAUI (or at least MAUI Blazor) have a minimum support of iOS 11 across the board, in which case we wouldn't want exclusions for older iOS's that we don't even support. Do you know how to configure that?

Reply:
I think that is it
https://github.com/dotnet/maui/blob/5d942a1abd03dfac29dc00d9970a17a01880cc60/Directory.Build.targets#L16-L19

Reply:
Ah perfect, thank you!

Reply:
@buyaa-n so then for this case I would remove the `if()` check and do a `#pragma` or something to ignore the warning because we will deal with it properly in https://github.com/dotnet/maui/issues/6078.

Reply:
If the API will not work without this API call then instead of platform check it should have the attribute to propagate the warning. But as it is overwritten method in case it is called by the parent instance where will be no warning

Reply:
@buyaa-n right, understood. If we just change the entire project to have `SupportedOSPlatformVersion = 12`, is that sufficient so that people won't accidentally try to use these types in projects that don't have at least the same `SupportedOSPlatformVersion` level? Or do we need to do that _and_ also annotate the types/members with the attributes?

Reply:
Project level attribute will be added to the assembly therefore any API from the assembly would be only for iOS 12 or above, if that is not expected I would add types/members level annotation 

There is several other places where overwritten API are annotated or suppressed I am planning to update/propagate those annotations into the Type/API level